straxen/contexts.py

```diff
@@ -77,6 +78,13 @@ def xenonnt_online(output_folder='./strax_data',
     st.register_all(have_nT_plugins)
     st.register([straxen.DAQReader, straxen.LEDCalibration])
 
+    st.set_config({
```

Comment:
Sorry I should have been more specific, the early commissioning context clones the config of the xenon_nt_online context. As such, could you also save the previous settings to that context in a similar fashion?

Reply:
Yep!